.github/workflows/publish-swagger.yml

```diff
@@ -0,0 +1,77 @@
+name: Publish Swagger
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    outputs:
+      swagger_valid: ${{ steps.validate-swagger.outputs.swagger_valid }}
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v3
+
+      - name: Build and Run Docker Compose
+        run: |
+          docker compose -f ${{ github.workspace }}/Server/docker-compose.yml up -d
```

Comment:
Can we maybe just bring up the services we need? Else we are introducing risk that one of the services fails and this pipeline fails. (looking at you Directus! 👀 )